test/runtests.jl

```diff
@@ -10,7 +10,7 @@ using Statistics
 using Test
 
 # For testing purposes we use a lot of
-using ChainRulesCore: extern, accumulate, accumulate!, store!, @scalar_rule,
+using ChainRulesCore: extern, @scalar_rule,
```

Comment:
```suggestion
using ChainRulesCore: extern, @scalar_rule, unthunk, canonicalize
```
Why are we not just doing `using ChainRulesCore` ?